chapters/en/slides/module6/module6_01.Rmd

```diff
@@ -0,0 +1,340 @@
+
+So far we have seen that beef is a major contributing factor to deforestation in Brazil,
+but there are many more aspect of the food production chain that we have not considered,
+such as the use of other resources
+
+Changing beef for other products would be highly beneficial,
+but how achievable it is 
+and what cost is associated with making the change
+we don't know from the data we have explored here.
+
+Let's continue our story by looking at the same breakdown for Indonesia,
+is beef production the biggest issue there as well?
+
+---
+
+## Palm oil is the biggest contributor to deforestation in Indonesia
+
+```{python}
+# TODO save wrangled data
+indonesia_drivers = pd.read_csv('data/deforestation-drivers-indonesia.csv').query('2013 > Year > 2003').groupby('Entity').sum().reset_index()
+alt.Chart(indonesia_drivers, title='Palm oil was the leading cause of deforestation between 2003 and 2013 in Indonesia').mark_bar().encode(
+    x='deforestation_hectares',
+    y=alt.Y('Entity', title='', sort='x'))
+```
+
+Notes:
+This slide shows the biggest drivers of deforestation in Indonesia
+between the years 2003 and 2013.
+
+Here,
+it is palm oil that drives most of the deforestation,
+although small scale clearings, timber plantation, and grasslands also contribute substantially.
+You might have read many negative things about palm oil already,
+as it has been in media's focus due to its link to tropical deforestation.
+
+From this chart alone it surely seems that decreasing palm oil production
+would lead to less deforestation.
+
+But if palm oil production would decrease
+while the world's demand for vegetable oils remains constant,
+another oil crop would need to take its place.
+
+Similar to what we did for beef,
+let's explore how efficient palm oil and its alternatives are
+when it comes to land use versus yield.
+
+---
+
+## Palm oil is the most efficient use of land for vegetable oil production
+
+```{python}
+oil_land_use = pd.read_csv('data/area-land-needed-to-global-oil.csv')
+alt.Chart(oil_land_use).mark_bar().encode(
+    alt.Y('Entity', title='', sort='x'),
+    alt.X('Area needed to meet global vegetable oil demand (ha)'))
+```
+
+Notes:
+This chart shows how many hectares 
+would be needed for the crops of common vegetable oils
+to meet the entire world's oil demands.
+
+It turns out that palm oil is the most efficient use of land
+for the amount of oil produced!
+If we wanted to replace it with e.g. olive oil,
+we would need almost 10x the land used.
+
+Just as with beef,
+the story of palm oil is more complex that we can show in a single visualization
+and it is not as easy as saying that we should use palm oil
+solely based on the criteria of efficiency.
+
+For example,
+the type of land where the oil crop grows matters.
+Replacing tropical forests containing rich biodiversity
+with palm oil plantages
+has a more severe impact
+than switching to other oil crops on already cultivated land elsewhere in the world.
```

Comment:
```suggestion
It turns out that palm oil is the most efficient use of land
for the amount of oil produced!
For example, if we wanted to replace it with olive oil,
we would need almost 10x the land to do so!

Just as with Brazil and it's use of pasture from deforested land, 
to raise cattle for beef beef, the story of palm oil is more complex 
then we can illustrate in a single visualization. 
```

Reply:
I think the last part here is important, what is the reason for removing it?

Reply:
Because what is being said is quite complex, and kind of goes against palm oil being more efficient - which we just showed in the plot. 

I think the problem is the ordering, now that I reflect more on it. Perhaps we should change it to this:

_It turns out that palm oil is the most efficient use of land
for the amount of oil produced!
For example, if we wanted to replace it with olive oil,
we would need almost 10x the land to do so!

So does it make sense for Indonesia to keep clearing
forests and creating more palm plantations because they
are the most efficient oil crop?

No, our conclusions cannot stop here. Things are not so 
simple. The type of land where the oil crop grows also matters.
Replacing tropical forests containing rich biodiversity
with palm oil plantations has a more severe, negative impact
on the environment compared to changing the type of 
oil crops being grown on land that has already been deforested 
and used for agriculture elsewhere in the world.

Just as with Brazil and it's use of pasture from deforested land, 
to raise cattle for beef beef, the story of palm oil is more complex 
then we can illustrate in a single visualization._

Reply:
Sounds great! Adding a mention about sustainble palm oil